### PR TITLE
Follow up: CIS Compliance guide fixes 

### PR DESCRIPTION
**Pull request by ina-stoyanova**
This PR is a follow up from the previous [CIS Compliance guide PR](https://github.com/gruntwork-io/gruntwork-io.github.io/pull/426) to address all the NITs and additional feedback. This should help us to manage the changes better and see them easier (the last one was getting too big).

The NITs are taken from comments in this PR and translated into these checkboxes below:

**Cleanup tasks:**
- [x] Change accounts declaration from `accounts = {x,y,z}` to `accounts = jsondecode(file("accounts.json"))` See [comment](https://github.com/gruntwork-io/gruntwork-io.github.io/pull/426#discussion_r652050290)
- [x] Update formatting of for-loops
- [x] Update `local.accounts["stage"]` to `local.accounts.stage` to be consistent across guides (e.g. the LZ guide) See [comment](https://github.com/gruntwork-io/gruntwork-io.github.io/pull/426#discussion_r652052836)
- [x] Review references of accounts such as in [this comment](https://github.com/gruntwork-io/gruntwork-io.github.io/pull/426#discussion_r652053437)
- [x] Remove specific regions mentioned - e.g. `us-east-1`, and replace with a generic placeholder. See [comment](https://github.com/gruntwork-io/gruntwork-io.github.io/pull/426#discussion_r652053938)
- [x] Replace `acme-*` specific placeholders with potentially `<RESOURCE_ACCOUNT>` like suggested [here ](https://github.com/gruntwork-io/gruntwork-io.github.io/pull/426#discussion_r652055340) and [here](https://github.com/gruntwork-io/gruntwork-io.github.io/pull/426#discussion_r652773187)
- [x] Replace `name_prefix = "stage-logs"` with something unique like `name_prefix = "<SOME UNIQUE IDENTIFIER>-logs"` - see [this comment](https://github.com/gruntwork-io/gruntwork-io.github.io/pull/426#discussion_r652064101)
- [x] Replace specific ARNs with placeholders - e.g. `kms_cmk_arn = "arn:aws:kms:us-east-1:${local.accounts.shared}:alias/ami-encryption"` should be more generic - See [this comment](https://github.com/gruntwork-io/gruntwork-io.github.io/pull/426#discussion_r652061073)
- [x] Remove `force_destroy_*` params - we don't need those in a production guide - See [this comment](https://github.com/gruntwork-io/gruntwork-io.github.io/pull/426#discussion_r652766413)
- [x] Set defaults for `max_session_duration_machine_users` and `max_session_duration_human_users` - See [this comment ](https://github.com/gruntwork-io/gruntwork-io.github.io/pull/426#discussion_r652768799)

**Readability tasks:**
More generic feedback to address (⚠️ Consider a separate PR for these if they're not small and quick changes)
- [x] A "very large number of sub-sections below deployment walkthrough -  See [this comment](https://github.com/gruntwork-io/gruntwork-io.github.io/pull/426#pullrequestreview-685262124) - potentially group into sections
- [x] Review the section "The Gruntwork solution" section a little hard to follow" and follow the suggestions by Jim - See [comment here](https://github.com/gruntwork-io/gruntwork-io.github.io/pull/426#discussion_r652755048)
- [x] Introduce a `common.hcl` to hold hard-coded values that are repeated across the code examples - See [this comment](https://github.com/gruntwork-io/gruntwork-io.github.io/pull/426#discussion_r652772822)
- [x] Call out explicitly that we have baselines for all types of accounts - root, logs, app and security - [See this comment](https://github.com/gruntwork-io/gruntwork-io.github.io/pull/426#discussion_r652777981)
- [x] Remove any mention of the `infra-modules` - See [this comment](https://github.com/gruntwork-io/gruntwork-io.github.io/pull/426#discussion_r652780292)
- [x] Transform the message around Security Hub showing 1.2 compliance in AWS console to be a `callout` format so it's easy for customers to spot - See [this comment](https://github.com/gruntwork-io/gruntwork-io.github.io/pull/426#discussion_r652781975)

**Potential bugs:** (⚠️ Consider a separate PR for these if they're not small and quick changes)
- [x] See if the right baseline is used in the `terraform source` on lines 978-979 - See [this comment](https://github.com/gruntwork-io/gruntwork-io.github.io/pull/426#discussion_r652767158) 
- [x] Does the code comment on lines 1093-1094 make sense - See [this comment](https://github.com/gruntwork-io/gruntwork-io.github.io/pull/426#discussion_r652767915)
- [x] Don't allow dev permissions in the root account - See [this comment](https://github.com/gruntwork-io/gruntwork-io.github.io/pull/426#discussion_r652768193) 
- [x] Do we need the `auto-deploy` permissions in the root baseline example? Remove it, as we shouldn't be needing this access - See [this comment](https://github.com/gruntwork-io/gruntwork-io.github.io/pull/426#discussion_r652768425)
- [x] We should remove the `dev_services` in the `logs` account example - See [this comment](https://github.com/gruntwork-io/gruntwork-io.github.io/pull/426#discussion_r652774922)
- [x] Review the `logs` account example - See [this comment](https://github.com/gruntwork-io/gruntwork-io.github.io/pull/426#discussion_r652775703) & [this one](https://github.com/gruntwork-io/gruntwork-io.github.io/pull/426#discussion_r652775837)
- [x] Remove `auto-scaling` configuration from `logs` account - do we need any? - See [this comment](https://github.com/gruntwork-io/gruntwork-io.github.io/pull/426#discussion_r652776023) 
- [x] Review Security account example code - See [this comment](https://github.com/gruntwork-io/gruntwork-io.github.io/pull/426#discussion_r652779627)

**Additional notes:**
- To make sure this PR is quick and easy, some of these tasks might need to be split out into separate PRs too
- To make sure all the [old PR comments](https://github.com/gruntwork-io/gruntwork-io.github.io/pull/426) are met - please review them one by one and assess if they still make sense to follow

### CODE REVIEW

**Comment by rhoboat** [comment]
Holy cannoli, that's a lot of stuff in the checklist. Can I help, since I've made a lot of these suggestions, and I've already made these changes in the LZ guide?

**Comment by rhoboat** [comment]
I took care of a few of these in #442 if that helps!

File: _posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc [hunk 1]
```diff
@@ -964,19 +1010,20 @@ Next, create a `terragrunt.hcl` file in `infrastructure-live`. It should go unde
 
 ----
 infrastructure-live
-  └ root
+  └ prod
```
Comment (marinalimeira) [inline _posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc:1013]: Not sure why the change? this needs to setup the root account before, `prod` references the production environment where the customer run their web apps.

File: _posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc [hunk 2]
```diff
@@ -993,14 +1040,14 @@ IMPORTANT: We **strongly** recommend setting Terraform parallelism to a low valu
...
+.infrastructure-live/prod/_global/account-baseline/terragrunt.hcl
 [source,hcl]
 ----
 locals {
-  aws_region = "us-east-1"
+  aws_region = local.aws_region
 
   accounts = {
-    root = "216044045972"
+    root = local.accounts.root
```
Comment (marinalimeira) [inline _posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc:1050]: this is a recursion 😂  it is referencing itself.

File: _posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc [hunk 3]
```diff
@@ -1108,12 +1149,6 @@ inputs = {
     "iam:GetRole",
     "iam:GetRolePolicy",
   ]
-
-  # Configures the auto deploy max session duration to be 4 hours.
```
Comment (marinalimeira) [inline _posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc:1112]: Why remove this code?

File: _posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc [hunk 4]
```diff
@@ -1553,26 +1575,6 @@ inputs = {
...
 
-  # By granting access to the root ARN of the Security account in each of the roles below,
-  # we allow administrators to further delegate access to other IAM entities
-
-  # A role that allows administrator access to the account.
-  allow_full_access_from_other_account_arns = [local.security_account_root_arn]
-
-  # A role for developers to use to access services in the account.
-  # Access to services is managed using the dev_permitted_services input.
-  allow_dev_access_from_other_account_arns = [local.security_account_root_arn]
```
Comment (marinalimeira) [inline _posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc:1564]: Why remove this examples?

File: _posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc [hunk 5]
```diff
@@ -2075,27 +1999,6 @@ inputs = {
...
   # Cross-account IAM role permissions
   ##################################
-
-  # By granting access to the root ARN of the Security account in each of the roles below,
-  # we allow administrators to further delegate access to other IAM entities
-
-  # A role that allows administrator access to the account.
-  allow_full_access_from_other_account_arns = [local.security_account_root_arn]
-
-  # A role for developers to use to access services in the account.
```
Comment (marinalimeira) [inline _posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc:2085]: Same here, why the removal?

File: _posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc [hunk 6]
```diff
@@ -944,13 +936,67 @@ https://www.schneier.com/blog/archives/2016/08/nist_is_no_long.html[no longer re
...
 
+[[deployment_approach]]
+=== Deployment approach
+Before we dive into the code and deployment for each of the resources, let's take a step back and understand how the code is structured.
+Most of the features explained in the "Production-grade design" section will be deployed by using the Landing Zone solution, and some more standalone modules like the VPC module.
+
+The Landing Zone will be deployed in three steps - the `account-baseline-root` to set up your organisation-wide configurations, and create the necessary child AWS accounts. Next, we'll need to apply the `account-baseline-app` baseline against the created logs account, which would in its turn set up the CloudTrail, AWS Config buckets and a few more settings that will be used for aggregation of logs and metrics from the whole organization. Then the `account-baseline-security` will be applied, and that's responsible to set up your IAM roles, groups that would allow you to access the rest of the accounts within your organization. And finally, the `account-baseline-app` will be applied to an AWS account with the purpose of hosting an application.
```
[the remark this answers is not in the document]
Comment (ina-stoyanova) [reply _posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc:944]: ah yes, of course! thanks 🙏

File: _posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc (hunk 1 above)
Comment (ina-stoyanova) [reply _posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc:1013]: oh, interesting. I looked at the `for-production` example code that I believe must have been generated by the RefArch 2.0 and didn't see `root` folder there - I could find all the relevant files in there. How come this one was called `root`?

File: _posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc (hunk 2 above)
Comment (ina-stoyanova) [reply _posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc:1050]: hahahha 😬 let's change it to <ROOT_ACCOUNT_ID>

File: _posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc (hunk 3 above)
Comment (ina-stoyanova) [reply _posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc:1112]: CIS 1.3 suggests not to have any roles and permissions with deployment, or full-access, or dev-access. The only roles that should be configured are ssh, readonly, billing & support. (See CIS 1.3 1.15-1.18)

File: _posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc (hunk 4 above)
Comment (ina-stoyanova) [reply _posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc:1564]: Well this is for the logs account, and we don't really need these dev roles in there (what do we expect it to be useful for?). Or at least that was my understanding. What do you think?

File: _posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc (hunk 5 above)
Comment (ina-stoyanova) [reply _posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc:2085]: CIS 1.3 suggests no full access here.

File: _posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc (hunk 1 above)
Comment (marinalimeira) [reply _posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc:1012]: Because we don't generate a `root` baseline in the Ref Arch

File: _posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc (hunk 2 above)
Comment (marinalimeira) [reply _posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc]: Ok, the previous IDs were random ones

File: _posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc (hunk 3 above)
Comment (marinalimeira) [reply _posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc:1112]: Ah, I see. I got this from the Ref Arch generated files

File: _posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc (hunk 1 above)
Comment (ina-stoyanova) [reply _posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc:1012]: Hm, ok. I'm happy to change it back - it was a little bit confusing for me, but I think your logic makes sense!

File: _posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc (hunk 2 above)
Comment (ina-stoyanova) [reply _posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc]: ahh! I didn't know that - they looked real. I'll actually replace it with `0123456789` to be on the safe side

**Comment by ina-stoyanova** [comment]
Ok, so I would like to merge this asap, otherwise, this is getting too big again. Is there any big blocker for this PR that is left for us to fix?

File: _posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc [hunk 7]
```diff
@@ -944,13 +936,66 @@ https://www.schneier.com/blog/archives/2016/08/nist_is_no_long.html[no longer re
...
+
+  # Extract the region for easy access
+  aws_region = local.region_vars.locals.aws_region
+}
+----
+
+You'll need to create these files in order to be able to follow the code examples following. For examples on what to put in each of these files, refer to the https://github.com/gruntwork-io/terraform-aws-cis-service-catalog/tree/master/examples/for-production[`terraform-aws-cis-service-catalog` repository]. The `locals` configuration above can be used in each of the `terragrunt.hcl` files to help you avoid repetition. Note that the examples below won't show this in the interest of clarity.
 
-Next, we'll configure a security baseline for the root account that is responsible for creating all the child accounts.
+Next, we'll configure the `account-baseline-root` baseline for the root account that is responsible for creating all the child accounts.
```
Comment (marinalimeira) [inline _posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc:998]: This phrase is a bit weird for me 🤔 

> the `account-baseline-root` baseline for the root account 

account, baseline and root appears two times each 😅  this is super NIT

File: _posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc [hunk 8]
```diff
@@ -944,13 +936,66 @@ https://www.schneier.com/blog/archives/2016/08/nist_is_no_long.html[no longer re
...
+
+First, let's set consider the repository structure that is recommended by this guide. It is available for your reference in the "/examples/for-production" section in the https://github.com/gruntwork-io/terraform-aws-cis-service-catalog/tree/master/examples/for-production[`terraform-aws-cis-service-catalog` repository]. Consider the following directory structure for your `infrastructure-live` repository. It showcases the configuration files foryour local variables.
+
+[source]
+----
+.
+└ infrastructure-live
+    └ root
+        └ account.hcl
+        └ global
```
Comment (marinalimeira) [inline _posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc:966]: Global here is missing an `_`
```suggestion
        └ _global
```

File: _posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc [hunk 9]
```diff
@@ -589,16 +588,16 @@ Use the link:https://www.terraform.io/docs/providers/aws/r/cloudtrail.html[`aws_
...
+cloud_watch_logs_group_arn    = local.cloudtrail_s3_bucket_arn
 
 event_selector {
   read_write_type           = "All"
   include_management_events = true
 
   data_resource {
     type   = "AWS::S3::Object"
-    values = [<YOUR BUCKET ARN>]
+    values = ["${local.cloudtrail_s3_bucket_name}"]
```
[the remark this answers is not in the document]
Comment (ina-stoyanova) [reply _posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc:600]: good catch!

File: _posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc (hunk 7 above)
Comment (ina-stoyanova) [reply _posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc:998]: ahhhh, readability is indeed awful 😰 , to thanks for point it out to me!

File: _posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc (hunk 8 above)
Comment (ina-stoyanova) [reply _posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc:966]: good catch too!

File: _posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc [hunk 10]
```diff
@@ -589,16 +588,16 @@ Use the link:https://www.terraform.io/docs/providers/aws/r/cloudtrail.html[`aws_
 is_multi_region_trail         = true
 include_global_service_events = true
 enable_log_file_validation    = true
-s3_bucket_name                = <YOUR BUCKET NAME>
-cloud_watch_logs_group_arn    = <YOUR CLOUDWATCH LOGS GROUP ARN>
+s3_bucket_name                = local.cloudtrail_s3_bucket_name
+cloud_watch_logs_group_arn    = local.cloudtrail_s3_bucket_arn
```
[the remark this answers is not in the document]
Comment (ina-stoyanova) [reply _posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc:592]: Good question - I assumed it's the same bucket. Where's best to find this..... 🤔  (i'm looking into this now)

[the remark this answers is not in the document]
Comment (ina-stoyanova) [reply _posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc:592]: I think we've got the relevant output in the cis repo `root` `outputs.tf` example - `cloudtrail_cloudwatch_group_arn`. I"m now referring to that one.

File: _posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc [hunk 11]
```diff
@@ -589,16 +588,16 @@ Use the link:https://www.terraform.io/docs/providers/aws/r/cloudtrail.html[`aws_
 is_multi_region_trail         = true
 include_global_service_events = true
 enable_log_file_validation    = true
-s3_bucket_name                = <YOUR BUCKET NAME>
-cloud_watch_logs_group_arn    = <YOUR CLOUDWATCH LOGS GROUP ARN>
+s3_bucket_name                = local.cloudtrail_s3_bucket_name
```
Comment (marinalimeira) [inline _posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc:591]: Reading [this section at the guide](https://deploy-preview-441--keen-clarke-470db9.netlify.app/guides/compliance/how-to-achieve-cis-benchmark-compliance/#logging-2), this `local` seems a bit  out of place, it is not connected to the Landing Zone steps so I think that ` cloud_watch_logs_group_arn    = <YOUR CLOUDWATCH LOGS GROUP ARN>` are more readable here.

File: _posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc [hunk 12]
```diff
@@ -944,14 +936,66 @@ https://www.schneier.com/blog/archives/2016/08/nist_is_no_long.html[no longer re
 https://www.theverge.com/2017/9/18/16328172/sms-two-factor-authentication-hack-password-bitcoin[vulnerabilities with the cellular system],
 so using a virtual or hardware MFA device is preferable; that said, MFA with SMS is still better than no MFA at all.
 
+[[deployment_approach]]
+=== Deployment approach
+Before we dive into the code and deployment for each of the resources, let's take a step back and understand how the code is structured.
```
Comment (marinalimeira) [inline _posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc:941]:
```suggestion
Before we dive into the code and deployment for each resource, let's take a step back and understand how the code is structured.
```

File: _posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc [hunk 13]
```diff
@@ -944,14 +936,66 @@ https://www.schneier.com/blog/archives/2016/08/nist_is_no_long.html[no longer re
...
+Most of the features explained in the <<production_grade_design>> section will be deployed by using the Landing Zone solution, and some more standalone modules like the VPC module.
```
Comment (marinalimeira) [inline _posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc:942]:
```suggestion
Most of the features explained in the <<production_grade_design>> section will be deployed using the Landing Zone solution, and some more standalone modules like the VPC module.
```

File: _posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc [hunk 14]
```diff
@@ -944,14 +936,66 @@ https://www.schneier.com/blog/archives/2016/08/nist_is_no_long.html[no longer re
...
+
+The Landing Zone will be deployed in three steps - the `account-baseline-root` to set up your organisation-wide configurations, create the necessary child AWS accounts, set up the CloudTrail and AWS Config buckets. Next, we'll need to apply the `account-baseline-app` against the created logs account, adding more settings that will be used for aggregation of logs and metrics from the whole organization. Then the `account-baseline-security` will be applied, and that's responsible to set up your IAM roles, groups that would allow you to access the rest of the accounts within your organization. And finally, the `account-baseline-app` will be applied to an AWS account with the purpose of hosting an application.
```
Comment (marinalimeira) [inline _posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc:944]:
```suggestion
The Landing Zone will be deployed in three steps - the `account-baseline-root` to set up your organization-wide configurations, create the necessary child AWS accounts, set up the CloudTrail and AWS Config buckets. Next, we'll need to apply the `account-baseline-app` against the created logs account, adding more settings that will be used for aggregation of logs and metrics from the whole organization. Then the `account-baseline-security` will be applied, and that's responsible to set up your IAM roles and groups that would allow you to access the rest of the accounts within your organization. And finally, the `account-baseline-app` will be applied to an AWS account with the purpose of hosting an application.
```

File: _posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc [hunk 15]
```diff
@@ -944,14 +936,66 @@ https://www.schneier.com/blog/archives/2016/08/nist_is_no_long.html[no longer re
...
+
+The standalone modules will follow the pattern of referencing the module and providing the necessary input variables for it, then applying with `terragrunt`.
 
-=== Configure the security baseline for the root account
+[[deploy_landingzone]]
+=== Deploy Landing Zone solution
+
+==== Apply the `account-baseline-root` to the root account
+
+===== Configure the `account-baseline-root` for the root account
```
Comment (marinalimeira) [inline _posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc:953]: I think we can remove this sub-level title. Just leave the one at L951

File: _posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc [hunk 16]
```diff
@@ -944,14 +936,66 @@ https://www.schneier.com/blog/archives/2016/08/nist_is_no_long.html[no longer re
...
 
 [.exceptional]
-IMPORTANT: You must be a [js-subscribe-cta]#Gruntwork subscriber# to access `terraform-aws-cis-service-catalog`.
+IMPORTANT: You must be a [js-subscribe-cta]#Gruntwork Compliance subscriber# to access the Gruntwork Infrastructure as Code Library and the https://github.com/gruntwork-io/terraform-aws-cis-service-catalog/[CIS AWS Foundations Benchmark modules].
+
+First, let's set consider the repository structure that is recommended by this guide. It is available for your reference in the "/examples/for-production" section in the https://github.com/gruntwork-io/terraform-aws-cis-service-catalog/tree/master/examples/for-production[`terraform-aws-cis-service-catalog` repository]. Consider the following directory structure for your `infrastructure-live` repository. It showcases the configuration files foryour local variables.
```
Comment (marinalimeira) [inline _posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc:958]:
```suggestion
First, let's set consider the repository structure that is recommended by this guide. It is available for your reference in the `/examples/for-production"` section in the https://github.com/gruntwork-io/terraform-aws-cis-service-catalog/tree/master/examples/for-production[`terraform-aws-cis-service-catalog` repository]. Consider the following directory structure for your `infrastructure-live` repository. It showcases the configuration files for your local variables.
```

File: _posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc [hunk 17]
```diff
@@ -944,14 +936,66 @@ https://www.schneier.com/blog/archives/2016/08/nist_is_no_long.html[no longer re
...
+        └ account.hcl
+        └ _global
+            └ region.hcl
+        └ us-east-1
+            └ region.hcl
+    └ common.hcl
+    └ accounts.json
+----
+
+Each of the `region.hcl`, `account.hcl` and `common.hcl` should contain the relevant information, so in your modules you're able to reference the values like this:
```
Comment (marinalimeira) [inline _posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc:974]:
```suggestion
Each of the `region.hcl`, `accounts.hcl` and `common.hcl` should contain the relevant information, so in your modules, you're able to reference the values like this:
```

File: _posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc [hunk 18]
```diff
@@ -944,14 +936,66 @@ https://www.schneier.com/blog/archives/2016/08/nist_is_no_long.html[no longer re
...
+
+  # Automatically load region-level variables
+  region_vars = read_terragrunt_config(find_in_parent_folders("region.hcl"))
+
+  # Extract the region for easy access
+  aws_region = local.region_vars.locals.aws_region
+}
+----
+
+You'll need to create these files in order to be able to follow the code examples following. For examples on what to put in each of these files, refer to the https://github.com/gruntwork-io/terraform-aws-cis-service-catalog/tree/master/examples/for-production[`terraform-aws-cis-service-catalog` repository]. The `locals` configuration above can be used in each of the `terragrunt.hcl` files to help you avoid repetition. Note that the examples below won't show this in the interest of clarity.
```
Comment (marinalimeira) [inline _posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc:996]:
```suggestion
You'll need to create these files to be able to follow the code examples following. For examples on what to put in each of these files, refer to the https://github.com/gruntwork-io/terraform-aws-cis-service-catalog/tree/master/examples/for-production[`terraform-aws-cis-service-catalog` repository]. Each of the `terragrunt.hcl` files will use the above `locals` configuration to help you avoid repetition. Note that the examples below won't show this in the interest of clarity.
```

NIT: Trying to remove passive voice. Not super sure about this change.

File: _posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc [hunk 19]
```diff
@@ -944,14 +936,66 @@ https://www.schneier.com/blog/archives/2016/08/nist_is_no_long.html[no longer re
...
+
+===== Configure the `account-baseline-root` for the root account
 
 [.exceptional]
-IMPORTANT: You must be a [js-subscribe-cta]#Gruntwork subscriber# to access `terraform-aws-cis-service-catalog`.
+IMPORTANT: You must be a [js-subscribe-cta]#Gruntwork Compliance subscriber# to access the Gruntwork Infrastructure as Code Library and the https://github.com/gruntwork-io/terraform-aws-cis-service-catalog/[CIS AWS Foundations Benchmark modules].
+
+First, let's set consider the repository structure that is recommended by this guide. It is available for your reference in the "/examples/for-production" section in the https://github.com/gruntwork-io/terraform-aws-cis-service-catalog/tree/master/examples/for-production[`terraform-aws-cis-service-catalog` repository]. Consider the following directory structure for your `infrastructure-live` repository. It showcases the configuration files foryour local variables.
+
+[source]
```
Comment (marinalimeira) [inline _posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc:960]: I don't think you need this `[source]` here. It rendered weird in the website, with some white lines crossing the text
Screen Shot 2021-07-07 at 18 19 49

File: _posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc [hunk 20]
```diff
@@ -997,10 +1042,10 @@ Set the variables for the `account-baseline-root` module in this environment in
 [source,hcl]
 ----
 locals {
-  aws_region = "us-east-1"
+  aws_region = local.aws_region
 
   accounts = {
-    root = "216044045972"
+    root = "<ROOT_ACCOUNT_ID>"
```
Comment (marinalimeira) [inline _posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc:1048]: So should this root account be in the `accounts.hcl`?

File: _posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc (hunk 3 above)
Comment (marinalimeira) [reply _posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc:1112]: I double checked this now and the default value for `max_session_duration_human_users` is 43200 (12 hours) and for `max_session_duration_machine_users` is 3600 (1 hour)

Comment (ina-stoyanova) [reply _posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc:1112]: yep 👍 I am not sure if you want me to add these back?

File: _posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc (hunk 11 above)
Comment (ina-stoyanova) [reply _posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc:591]: ok, changing it now! 🙇

File: _posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc (hunk 18 above)
Comment (ina-stoyanova) [reply _posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc:996]: That's ok! Thanks

File: _posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc (hunk 19 above)
Comment (ina-stoyanova) [reply _posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc:960]: hm, weird. Removed.

File: _posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc (hunk 20 above)
Comment (ina-stoyanova) [reply _posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc:1048]: I think we can leave it like this, since we ask them to create the root account manually first.

**Comment by ina-stoyanova** [comment]
Wohoo! Thanks for all the attention and feedback, @marinalimeira! Last round done! I hope to get this merged as soon as we can, so I guess if any other changes come up, I'm sure we can do separately. Thoughts?

**Review by yorinasub17 — APPROVED** [review]
LGTM!

**Comment by ina-stoyanova** [comment]
Yay! Thanks, Yori ❤️
